packages/common/src/templates/iconsLight.ts

```diff
@@ -85,11 +87,11 @@ export default function getIcon(theme: TemplateType) {
     case cxjs.name:
       return CxJSIconLight;
     case reason.name:
-      return ReasonIconLight;
+      return ReasonIconLigridsomeght;
```

Comment:
```suggestion
      return ReasonIconLight;
```

Reply:
Resolved in the commit that i am preparing for merge, used this to see which checks is this going to trigger since linting on gitpod (IDE used for this contribution) doesn't seem to catch everything (to get an image about how the logic is implemented to avoid regression)

Thanks for review though appreciated